unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NotificationMessagesHUD/ChatNotificationController.cs

```diff
@@ -182,16 +194,25 @@ private async UniTaskVoid AddNotificationAsync(ChatMessage message, Channel chan
                     body = await profanityFilter.Filter(message.body, cancellationToken);
                 }
 
-                var publicModel = new PublicChannelMessageNotificationModel(message.messageId,
-                    body, channel?.Name ?? message.recipient, channel?.ChannelId, message.timestamp,
-                    isMyMessage, senderName);
+                var publicModel = new PublicChannelMessageNotificationModel(
+                    message.messageId,
+                    body,
+                    channel?.Name ?? message.recipient,
+                    channel?.ChannelId,
+                    message.timestamp,
+                    isMyMessage,
+                    senderName,
+                    isOwnPlayerMentioned);
 
                 mainChatNotificationView.AddNewChatNotification(publicModel);
 
                 if ((string.IsNullOrEmpty(message.recipient) && openedChatId != ChatUtils.NEARBY_CHANNEL_ID)
                     || (!string.IsNullOrEmpty(message.recipient) && openedChatId != message.recipient))
                     if (topNotificationPanelTransform.Get().gameObject.activeInHierarchy)
                         topNotificationView.AddNewChatNotification(publicModel);
+
+                if (isOwnPlayerMentioned)
+                    AudioScriptableObjects.ChatReceiveMentionEvent.Play(true);
```

Comment:
controllers should not play sounds.. views should do it instead

Reply:
alright, makes sense